pkg/keycloak/proxy/server.go

```diff
@@ -1006,6 +1006,14 @@ func (r *OauthProxy) Run() error {
 	return nil
 }
 
+// Shutdown finishes the proxy service with gracefully period
+func (r *OauthProxy) Shutdown() {
+	gracefulPeriod := r.Config.ServerGraceTimeout
+	ctx, cancel := context.WithTimeout(context.Background(), time.Duration(time.Second*gracefulPeriod))
```

Comment:
there should not be time.Duration() only gracefulPeriod as it is already variable of type time.Duration

Reply:
My bad! Fixed.